src/math/exp_fcn_hifi.c

```diff
@@ -318,10 +383,13 @@ int32_t exp_fixed(int32_t x)
 	/* sofm_exp_int32() input is Q4.28, while x1 is Q5.27
 	 * sofm_exp_int32() output is Q9.23, while y0 is Q12.20
 	 */
-	y0 = Q_SHIFT_RND(sofm_exp_int32(Q_SHIFT_LEFT(xs, 27, 28)), 23, 20);
+	y0 = exp_hifi_q_shift_rnd(
+							  sofm_exp_int32(
+							  exp_hifi_q_shift_left(xs, 27, 28)),
+							  23, 20);
 	y = ONE_Q20;
 	for (i = 0; i < (1 << n); i++)
-		y = (int32_t)Q_MULTSR_32X32((int64_t)y, y0, 20, 20, 20);
+		y = (int32_t)exp_hifi_q_multsr_32x32((int64_t)y, y0, 20, 20, 20);
```

Comment:
this patch is not related with title(exp), you can't use one PR to cover all the changes, please move out this patch from this PR.